### PR TITLE
Add descriptions for capi models

### PR DESCRIPTION
**Pull request by kkudose**

### CODE REVIEW

File: packages/carrier-api/src/app/carrier-app-definition.ts [hunk 1]
```diff
@@ -36,49 +36,49 @@ export interface CarrierAppDefinition {
   ) => RegisterResponse | Promise<RegisterResponse>;
   /**
    * @description This method is used to create a label
-   * @param request
+   * @param request Data needed by the provider to create a label
```
Comment (dougwile) [inline packages/carrier-api/src/app/carrier-app-definition.ts:39]:
```suggestion
   * @param The request data needed by the provider to create a label
```

File: packages/carrier-api/src/models/addresses/pudo-location.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { AddressBase } from "./address-base";
 
+// TODO
```
Comment (dougwile) [inline packages/carrier-api/src/models/addresses/pudo-location.ts:3]: TODO needs addressing

File: packages/carrier-api/src/models/fulfillment-plans/fulfillment-plan-items.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { Identifier } from "../identifier";
 
+// TODO
```
Comment (dougwile) [inline packages/carrier-api/src/models/fulfillment-plans/fulfillment-plan-items.ts:3]: TODO needs addressing

File: packages/carrier-api/src/models/identifier.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+// todo
```
Comment (dougwile) [inline packages/carrier-api/src/models/identifier.ts:1]: TODO needs addressing

File: packages/carrier-api/src/models/labels/label-package.ts [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import { Identifier } from "../identifier";
 
+// TODO
```
Comment (dougwile) [inline packages/carrier-api/src/models/labels/label-package.ts:3]: TODO needs addressing

File: packages/carrier-api/src/models/pickup/cancellation-details.ts [hunk 6]
```diff
@@ -1,7 +1,9 @@
 import { CancellationReason } from "./cancellation-reason";
 
+/** @description Basic structure for a cancellation */
 export interface CancellationDetails {
   reason: CancellationReason;
+  // TODO
```
Comment (dougwile) [inline packages/carrier-api/src/models/pickup/cancellation-details.ts:6]: TODO needs addressing

File: packages/carrier-api/src/models/pickup/pickup-location-details.ts [hunk 7]
```diff
@@ -1,7 +1,9 @@
 import { ShipFrom } from "../addresses/ship-from";
 
+/** @description Location details for pickup */
 export interface PickupLocationDetails {
   pickup_address?: ShipFrom;
   location_notes?: string;
+  // todo
```
Comment (dougwile) [inline packages/carrier-api/src/models/pickup/pickup-location-details.ts:7]: TODO needs addressing

File: packages/carrier-api/src/models/pickup/pickup-shipment-details.ts [hunk 8]
```diff
@@ -1,6 +1,8 @@
 import { PickupShipment } from "./pickup-shipment";
 
+/** @description Details for a pickup shipment */
 export interface PickupShipmentDetails {
   pickup_service_code?: string;
+  // todo
```
Comment (dougwile) [inline packages/carrier-api/src/models/pickup/pickup-shipment-details.ts:6]: TODO needs addressing

File: packages/carrier-api/src/models/tracking/tracking-attribute.ts [hunk 9]
```diff
@@ -1,3 +1,4 @@
+// todo
```
Comment (dougwile) [inline packages/carrier-api/src/models/tracking/tracking-attribute.ts:1]: TODO needs addressing

File: packages/carrier-api/src/models/tracking/tracking-identifier.ts [hunk 10]
```diff
@@ -1,3 +1,4 @@
+// todo
```
Comment (dougwile) [inline packages/carrier-api/src/models/tracking/tracking-identifier.ts:1]: TODO needs addressing

File: packages/carrier-api/src/requests/create-label-request.ts [hunk 11]
```diff
@@ -16,27 +16,34 @@ import {
...
   ship_to: ShipTo;
   ship_from: ShipFrom;
   pickup_location?: PudoLocation;
   ship_from_display?: ShipFromDisplay;
+  /** @description Whether the shipment requires next day shipping */
   next_day?: boolean;
+  /** @description Whether the shipment is international */
   international?: boolean;
   reference?: string;
+  // todo
```
Comment (dougwile) [inline packages/carrier-api/src/requests/create-label-request.ts:46]: TODO needs addressing

**Review by dougwile — CHANGES_REQUESTED** [review]

File: packages/carrier-api/src/models/addresses/pudo-location.ts (hunk 2 above)
Comment (dougwile) [reply packages/carrier-api/src/models/addresses/pudo-location.ts:3]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Addresses/PudoLocation.cs

File: packages/carrier-api/src/app/carrier-app-definition.ts (hunk 1 above)
Comment (kkudose) [reply packages/carrier-api/src/app/carrier-app-definition.ts:39]: `request` is the name of the param (see line 35, 42, etc) and I'm following the convention [Justin used here](https://github.com/ShipEngine/connect/blob/kkudose/update-connect-capi-descriptions/packages/order-source-api/src/app/ordersource-app-definition.ts#L33)

File: packages/carrier-api/src/models/addresses/pudo-location.ts (hunk 2 above)
Comment (dougwile) [reply packages/carrier-api/src/models/addresses/pudo-location.ts:3]: can just reuse the descriptions from the api definition

File: packages/carrier-api/src/models/fulfillment-plans/fulfillment-plan-items.ts (hunk 3 above)
Comment (dougwile) [reply packages/carrier-api/src/models/fulfillment-plans/fulfillment-plan-items.ts:3]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/FulfillmentPlans/FulfillmentPlanItems.cs

File: packages/carrier-api/src/models/identifier.ts (hunk 4 above)
Comment (dougwile) [reply packages/carrier-api/src/models/identifier.ts:1]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Identifier.cs

File: packages/carrier-api/src/models/labels/label-package.ts (hunk 5 above)
Comment (dougwile) [reply packages/carrier-api/src/models/labels/label-package.ts:3]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Labels/LabelPackage.cs

File: packages/carrier-api/src/models/pickup/cancellation-details.ts (hunk 6 above)
Comment (dougwile) [reply packages/carrier-api/src/models/pickup/cancellation-details.ts:6]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Request/CancelPickup/CancellationDetails.cs

File: packages/carrier-api/src/models/pickup/pickup-location-details.ts (hunk 7 above)
Comment (dougwile) [reply packages/carrier-api/src/models/pickup/pickup-location-details.ts:7]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Pickup/PickupLocationDetails.cs

File: packages/carrier-api/src/models/pickup/pickup-shipment-details.ts (hunk 8 above)
Comment (dougwile) [reply packages/carrier-api/src/models/pickup/pickup-shipment-details.ts:6]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Pickup/PickupShipmentDetails.cs

File: packages/carrier-api/src/models/tracking/tracking-attribute.ts (hunk 9 above)
Comment (dougwile) [reply packages/carrier-api/src/models/tracking/tracking-attribute.ts:1]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Tracking/TrackingAttribute.cs

File: packages/carrier-api/src/models/tracking/tracking-identifier.ts (hunk 10 above)
Comment (dougwile) [reply packages/carrier-api/src/models/tracking/tracking-identifier.ts:1]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Models/Tracking/TrackingIdentifier.cs

File: packages/carrier-api/src/requests/create-label-request.ts (hunk 11 above)
Comment (dougwile) [reply packages/carrier-api/src/requests/create-label-request.ts:46]: https://github.com/shipstation/integrations-shipping/blob/master/carrierapi-common/CarrierApi.Common/Request/CreateLabelRequest.cs#L168

Comment (dougwile) [reply packages/carrier-api/src/requests/create-label-request.ts:46]: this bad boy is deprecated. We have a follow on ticket to actually just remove deprecated fields in CAPI from SEC so you can ignore this one

**Review by dougwile — APPROVED** [review]

**Pull request merged**